packages/paywall/example/src/profile.tsx

```diff
@@ -0,0 +1,53 @@
+import { useAccount, useConnect, useDisconnect } from 'wagmi'
+import { InjectedConnector } from 'wagmi/connectors/injected'
+import { Paywall } from '../../'
+import { networks } from '@unlock-protocol/networks'
+const paywall = new Paywall(networks)
+
+export function Profile() {
+  const { address, isConnected } = useAccount()
+  const { connect } = useConnect({
+    connector: new InjectedConnector({
+      options: {
+        name: 'Unlock Paywall Provider',
+        getProvider: () => {
+          const provider = paywall.getProvider('http://localhost:3000')
+          paywall.connect(provider)
```

Comment:
That's not rigth. The call to `connect` is internal to `Paywall` and will be performed (indirectly) when wagmi's `connect` is called (it calls eth_requestAccounts, which the Paywall "intercepts" to call its own internal `connect`).